samples/boards/mec15xxevb_assy6853/power_management/src/power_mgmt.c

```diff
@@ -18,7 +18,7 @@ LOG_MODULE_REGISTER(pwrmgmt_test);
 /* Thread properties */
 #define TASK_STACK_SIZE           1024ul
 #define PRIORITY                  K_PRIO_COOP(5)
-/* Sleep time should be lower than CONFIG_PM_MIN_RESIDENCY_SLEEP_1 */
+/* Sleep time should be lower than SUSPEND_TO_IDLE state */
```

Comment:
need to reword this, previous it was talking about residency, now it is talking about state.